src/PlayFabBuddy.Cli/Commands/Player/DeletePlayersBySegmentCommandSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel;
+using PlayFabBuddy.Lib.Interfaces.Adapter;
+using Spectre.Console.Cli;
+
+namespace PlayFabBuddy.Cli.Commands.Player;
+
+public class DeletePlayersBySegmentCommandSettings : PlayerSettings
+{
+    [Description(
+        $"The name of the segment Accounts should be deleted from. Use \"{IPlayStreamAdapter.AllPlayersSegmentName}\" to delete all Players.")]
```

Comment:
Can we add the default segments into the description or provide samples?

Reply:
Was thinking the same. added them in the docs and as constants. Perhaps add it to the Description annotation to put the constants to actual use?